tools/editor/plugins/script_editor_plugin.cpp

```diff
@@ -1364,8 +1364,6 @@ void ScriptEditor::_update_script_colors() {
 
 	bool enabled = EditorSettings::get_singleton()->get("text_editor/script_temperature_enabled");
 	bool highlight_current = EditorSettings::get_singleton()->get("text_editor/highlight_current_script");
-	if (!enabled)
```

Comment:
In the case where `script_temperature_enabled` is off, isn't it a bit overkill to have all those computations done just to change the colour of the current tab?

I think it might be clearer to either:
- change this line for an `if (enabled)` (and maybe also rename the var to `temperature_enabled` for clarity), adapt indentation accordingly for the whole block, and move the stuff relative to the current script highlight after the if block, or
- duplicate the logic for highlight_current in this `if (!enabled)` to highlight `tab_container->get_current_tab()`.

The first solution is likely clearer.